app/assets/stylesheets/semantic-ui/globals/_variables.scss

```diff
@@ -1,4 +1,37 @@
-$import-google-fonts: true !default;
-$font-url: 'https://fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic&subset=latin,latin-ext' !default;
-$font-name: 'Lato' !default;
-$font-family: $font-name, 'Helvetica Neue', Arial, Helvetica, sans-serif !default;
+$font-url: 'proxima-nova';
+$import-google-fonts: 'proxima-nova';
+$font-name: 'proxima-nova' !default;
+$font-family: $font-name, 'proxima-nova', Arial, Helvetica, sans-serif !default;
+
+$text-shadow: 0 1px 3px rgba(0,0,0,.5);
+
+$button-shadow: 0 3px 6px rgba(0,0,0,.25);
+$button-padding: 0.875em 1.5em 0.875em;
+$button-radius: 0.425rem;
+$button-border: 1px solid rgba(255, 255, 255, 0.15);
+$button-border-hover: 1px solid rgba(255, 255, 255, 0.4);
+
+$input-bg: rgba(0, 0, 0, 0.4);
+$input-padding: 0.685em 1em 0.685em;
+$input-inverted-placeholder: rgba(0,0,0,.4);
+
+$button-primary: linear-gradient(#4DB1D8, #1274B2);
```

Comment:
I would argue that adding `-color` is confusing as it is treated as a background-image. I would prefer `-gradient` or `-image` to avoid someone using `background-color: $button-primary`